src/client/utils/tracking/tracking.ts

```diff
@@ -69,7 +69,19 @@ export const { TrackAction, IdentifyAction } = setupTrackers<
   { debug: process.env.NODE_ENV === 'development' },
 )
 
-const adtractionProductMap: { [type in TypeOfContract]: number } = {
+const NOT_VALID_CONTRACTS = [
+  TypeOfContract.DkAccident,
+  TypeOfContract.DkAccidentStudent,
+  TypeOfContract.DkTravel,
+  TypeOfContract.DkTravelStudent,
+] as const
```

Comment:
Oh actually we should add all DK contract types
```suggestion
const NOT_VALID_CONTRACTS = [
  TypeOfContract.DkHomeContentOwn,
  TypeOfContract.DkHomeContentRent,
  TypeOfContract.DkHomeContentStudentOwn,
  TypeOfContract.DkHomeContentStudentRent,
  TypeOfContract.DkAccident,
  TypeOfContract.DkAccidentStudent,
  TypeOfContract.DkTravel,
  TypeOfContract.DkTravelStudent,
] as const
```

Reply:
okej we should remove the existing danish types as well?